### DISCUSSION TITLE
stream() and output() write black background

### DISCUSSION THREAD

**Discussion by geertbe** [dompdf/dompdf#2735] [Q&A]
Hello,
I'm getting a PDF with a black background when using the stream() and output() methods.
The html I'm loading with loadHtmlFile is clean.
Any idea?
Thanks, 
Geert

**Comment by bsweeney** [comment]
Are you saying that you are using both methods in the same script? Or that regardless of which one you use you are getting a black background? Can you provide a sample of the HTML + CSS and of the rendered output?

**Reply by geertbe** [reply]
Hello I meant I tried both methods, yet not at the same time. When reading your reply, I figured it wasn't just a black background, it looked like darkmode: white text on a black background.
I was using the default stylesheet from my website which has darkmode coding after the default body coding. I don't know if it had something to do. In any case, using that stylesheet was a bad idea.
I now use a stripped down "lightweight" css only for the PDF creation and it works like a charm.
Many thanks.

**Reply by bsweeney** [reply]
Might have been something with the selectors. If you want to share a sample of how dark mode is configured we can take a look. Otherwise, glad you were able to work around the issue.
